packages/dart_frog_cli/CONTRIBUTING.md

```diff
@@ -0,0 +1,134 @@
+
+The Dart Frog CLI is a [`CommandRunner`](https://pub.dev/documentation/args/latest/command_runner/CommandRunner-class.html) that exposes four visible commands:
+
+- `build`: Creates a production build by using the [`dart_frog_prod_server`](../../bricks/dart_frog_prod_server/) mason template to generate files.
+- `create`: Creates a new Dart Frog app by using the [`dart_frog_new`](../../bricks/dart_frog_new/) mason template to generate the files.
+- `dev`: Runs a local development server at a given port and listens to file changes, generating new files using the [`dart_frog_dev_server`](../../bricks/dart_frog_dev_server/) mason template on updates.
+- `update`: Updates the Dart Frog CLI's if possible.
+
+💡 **Note**: Dart Frog CLI's completion functionality is powered by [CLI Completion](https://github.com/VeryGoodOpenSource/cli_completion).
+
+## Opening an issue
+
+We highly recommend [creating an issue][bug_report_link] if you have found a bug, want to suggest a feature, or recommend a change. Please do not immediately open a pull request. Opening an issue first allows us to reach an agreement on a fix before you put significant effort into a pull request.
+
+When reporting a bug, please use the built-in [Bug Report][bug_report_link] template and provide as much information as possible including detailed reproduction steps. Once one of the package maintainers has reviewed the issue and we reach an agreement on the fix, open a pull request.
+
+## Developing for Dart Frog CLI
+
+To develop for Dart Frog CLI you will need to become familiar with Very Good Ventures processes and conventions:
+
+### Setting up your local development environment
+
+1. Install a valid [Dart SDK](https://dart.dev/get-dart) in your local environment. Compatible Dart SDK versions with test optimizer can be found [here](https://github.com/VeryGoodOpenSource/very_good_cli/blob/main/pubspec.yaml). If you have Flutter installed, you likely have a valid Dart SDK version already installed.
+
+2. Install [Very Good CLI](https://github.com/VeryGoodOpenSource/very_good_cli):
+
+```sh
+# 💻 Install Very Good CLI globally:
+dart pub global activate very_good_cli
+```
+
+3. Install all Dart Frog CLI's dependencies:
+
+```sh
+# 📂 Get project dependencies recursively with Dart Frog CLI's
+very_good packages get -r --ignore="bricks/**"
+```
+
+3. Run all Dart Frog CLI's tests:
+
+```sh
+# 🧪 Run Dart Frog CLI's unit tests (from packages/dart_frog_cli)
+dart test
+
+# 🧪💻 Run Dart Frog CLI's end to end test (from packages/dart_frog_cli/e2e)
+dart test
+```
+
+If some tests do not pass out of the box, please submit an [issue](https://github.com/VeryGoodOpenSource/dart_frog/issues/new/choose).
+
+4. Install your own version of Dart Frog CLI in your local environment:
+
+```sh
+# 🚀 Activate your own local version of Dart Frog CLI (from packages/dart_frog_cli)
+dart pub global activate --source path .
+```
+
+5. If you are modifying any templates, make sure to bundle them before activating:
```

Comment:
Should we specify where those templates are located at?

Reply:
If you think we should -- it can't hurt!